### PR TITLE
fix(sidebar): Sidebar nav button tooltips should display on hover

### PR DESCRIPTION
**Pull request by jstoffan**
Tooltips

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jstoffan has signed the CLA. Thanks for the pull request!

File: src/elements/content-sidebar/SidebarNavButton.js [hunk 1]
```diff
@@ -22,12 +22,12 @@ const SidebarNavButton = ({ children, interactionTarget, onNavigate, sidebarView
...
-                {({ match }) => {
-                    const isToggle = !!match && match.isExact;
+        <Route path={sidebarPath}>
+            {({ match }) => {
+                const isToggle = !!match && match.isExact;
 
-                    return (
+                return (
+                    <Tooltip position="middle-left" text={tooltip}>
                         <NavButton
```
Comment (priyajeet) [inline src/elements/content-sidebar/SidebarNavButton.js]: NavButton also has <Route> in it, should tooltip be inside there around the actual component?

Comment (jstoffan) [reply src/elements/content-sidebar/SidebarNavButton.js]: It worked as-is when I tested it locally. The NavButton component forwards all its props to its child, which may explain why it works properly.

**Comment by jstoffan** [comment]
I'll put adding an e2e test on the todo list, but I think we have other work to address at the moment. This will at least fix the bug in the immediate term.

**Pull request merged**